pkg/neonvm/controllers/vmmigration_controller.go

```diff
@@ -357,10 +362,22 @@ func (r *VirtualMachineMigrationReconciler) Reconcile(ctx context.Context, req c
 						Reason:  "Reconciling",
 						Message: message,
 					})
-				// finally update migration phase to Running
+				return r.updateMigrationStatus(ctx, migration)
+			case vmv1.VmMigrating:
+				// migration is in progress so we can scale CPU using sysfs
```

Comment:
> migration is in progress so we can scale CPU using sysfs

This should be forbidden because we only scale CPUs when the VM is in the `Scaling` phase (although eventually we may change that...)

Reply:
I think here I meant that we can call the neonvm-daemon to enable all required cpus, because before that machine wasn't yet booted and the neonvm was not accessible